CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Breaking Changes
   * Make `shippingMethod` property on `BTThreeDSecureRequest` an enum instead of a string
   * Remove `BTTokenizationService`
+  * Make `BTPaymentMethodNonceParser` private
```

Comment:
What do you all think of listing classes made private as "removed" in the CHANGELOG? That's what we've been doing in Android, since from an integration perspective the class does not exist anymore.

Reply:
Yeh - we talked about this for a bit. We were thinking it could potentially confuse our future-selves/ SDK team members if we say "remove" instead of "make private".